packages/cli/src/runner/crawler-command-runner.ts

```diff
@@ -25,7 +27,7 @@ export class CrawlerCommandRunner implements CommandRunner {
 
         console.log(`Crawling and scanning page ${scanArguments.url}`);
 
-        const scanResult = await this.crawlerEntryPoint.crawl({
+        this.scanResult = await this.crawlerEntryPoint.crawl({
```

Comment:
Why not just pass this as a parameter to the generate summary reports method? Would likely help with a future re-factor as well, if we decide to break apart this class.